src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -199,6 +199,24 @@ public static void AssertFormat(string messageFormat, params object[] data)
             }
         }
 
+        public static void AssertFormat(bool condition, string messageFormat, params object[] data)
```

Comment:
This method should be renamed to just `Assert`.  It's ok and expected to have multiple overloads of a method where only the number/type of parameters vary.  I.e. `Console.WriteLine`